src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((string) => string.length > 0)
+    .map((string) => string.split(':').map((part) => part.trim()))
+    .filter((element) => element[0] && element[1])
```

Comment:
The filter condition `element[0] && element[1]` ensures that both key and value are present. However, consider adding a check to ensure that `element[0]` and `element[1]` are not just empty strings after trimming.